### PR TITLE
localed: check both vc_keymap and X11 before returning early

### PR DESCRIPTION
**Pull request by YHNdnzj**
Currently, if vc_keymap is unchanged, localed just returns without modifying anything. However, the X11 part may need updating too. So we should check for both and ensure they are unmodified.

### CODE REVIEW

**Review by yuwata — CHANGES_REQUESTED** [review]
Thank you for working on this, and I am sorry that recently I am working so slowly...

I am slowly working on the issue by a kind of more comprehensive way.

File: src/locale/localed.c [hunk 1]
```diff
@@ -430,15 +415,33 @@ static int method_set_vc_keyboard(sd_bus_message *m, void *userdata, sd_bus_erro
                         return sd_bus_error_set_errnof(error, r, "Failed to convert keymap data: %m");
                 }
 
-                /* save the result of conversion to emit changed properties later. */
-                convert = r > 0;
+                x11_modified = r > 0; /* save the result of conversion to emit changed properties later. */
```
Comment (yuwata) [inline src/locale/localed.c:418]: This causes security issue, as `vconsole_convert_to_x11()` updates `Context`. So, with this change, unprivileged users can update `Context` without any authentications.

**Comment by YHNdnzj** [comment]
@yuwata Should I continue working on this or just leave it to you?

**Comment by yuwata** [comment]
> @yuwata Should I continue working on this or just leave it to you?

Today, I worked on the issue, and already finished half way. I can easily guess your work and mine mostly duplicated. So, unless you really want to continue this, please wait my PR for a while.

**Comment by YHNdnzj** [comment]
> Today, I worked on the issue, and already finished half way. I can easily
guess your work and mine mostly duplicated. So, unless you really want to
continue this, please wait my PR for a while.

I guess I’ll wait then :)

**Comment by yuwata** [comment]
-> #26219.

**Pull request closed without merging**